crates/containerd-shim-wasm/src/sandbox/instance.rs

```diff
@@ -22,7 +22,7 @@ pub struct InstanceConfig {
     bundle: PathBuf,
     /// Namespace for containerd
     namespace: String,
-    // /// GRPC address back to main containerd
+    /// GRPC address back to main containerd
```

Comment:
technically this should be a ttrpc address

Reply:
Hmm, I think this is indeed the grpc address. The one that the wasm-oci code uses to talk back with c8d.
IIRC, this is /var/containerd/containerd.sock, while the ttrpc socket is /var/containerd/containerd.sock.ttrpc

Reply:
Ah yes you're right! I merged this PR in but we can revert it back